__main__.py

```diff
@@ -0,0 +1,216 @@
+    self.passwords = []
+    self.recipients = []
+    self.recipfiles = []
+    self.outfile = []
+    self.identities = []
+    self.padding = "5"
+    self.armor = None
+    self.paste = None
+    self.debug = None
+
+
+encargs = dict(
+  askpass='-p --passphrase'.split(),
+  passwords='--password'.split(),
+  wideopen='--wide-open'.split(),
+  recipients='-r --recipient'.split(),
+  recipfiles='-R --keyfile --recipients-file'.split(),
+  identities='-i --identity'.split(),
+  outfile='-o --out --output'.split(),
+  armor='-a --armor'.split(),
+  paste='-A'.split(),
+  padding='--pad --padding'.split(),
+  debug='--debug'.split(),
+)
+
+decargs = dict(
+  askpass='-p --passphrase'.split(),
+  passwords='--password'.split(),
+  identities='-i --identity'.split(),
+  outfile='-o --out --output'.split(),
+  paste='-A'.split(),
+  debug='--debug'.split(),
+)
+
+benchargs = dict(debug='--debug'.split(),)
+
+
+def argparse():
+  # Custom parsing due to argparse module's limitations
+  av = sys.argv[1:]
+  if not av or any(a.lower() in ('-h', '--help') for a in av):
+    first, rest = cmdhelp.rstrip().split('\n', 1)
+    if stdout.isatty():
+      print(f'\x1B[1;44m{first:78}\x1B[0m\n{rest}')
+    else:
+      print(f'{first}\n{rest}')
+    sys.exit(0)
+  if any(a.lower() in ('-v', '--version') for a in av):
+    print(cmdhelp.split('\n')[0])
+    sys.exit(0)
+  ad = {}
+  args = Args()
+  # Support a few other forms for Age etc. compatibility (but only as the first arg)
+  if av[0].lower() in ('enc', 'encrypt', '-e', '--encrypt'):
+    args.mode, ad, modehelp = 'enc', encargs, f"{hdrhelp}\nEncryption options:\n{enchelp}"
+  elif av[0].lower() in ('dec', 'decrypt', '-d', '--decrypt'):
+    args.mode, ad, modehelp = 'dec', decargs, f"{hdrhelp}\nEncryption options:\n{enchelp}"
+  elif av[0].lower() in ('bench', 'benchmark'):
+    args.mode, ad, modehelp = 'benchmark', benchargs, f"{hdrhelp}"
+  elif next((k for k, v in encargs.items() if av[0] in v), None) is not None:
+    args.mode, ad, modehelp = 'enc', encargs, enchelp
+    av.insert(0, None)
+  else:
+    stderr.write(' 💣  Invalid or missing command (enc/dec/benchmark).\n')
+    sys.exit(1)
+
+  aiter = iter(av[1:])
+  longargs = [flag[1:] for switches in ad.values() for flag in switches if flag.startswith("--")]
+  shortargs = [flag[1:] for switches in ad.values() for flag in switches if not flag.startswith("--")]
+  print(shortargs)
+  for a in aiter:
+    aprint = a
+    if not a.startswith('-'):
+      args.files.append(a)
+      continue
+    if a == '-':
+      args.files.append(True)
+      continue
+    if a == '--':
+      args.files += args
+      break
+    if a.startswith('--'):
+      a = a.lower()
+    if not a.startswith('--') and len(a) > 2:
+      if a not in longargs:
+        if any(longarg[1:] in a for longarg in longargs) is True:
+          stderr.write(f' 💣  Multiple arguments cannot have long arguments: covert {args.mode} {a}\n')
+          sys.exit(1)
```

Comment:
Should not test for this. `-out` should be allowed and it means `-o -u -t`, not `--out`.